pkg/apis/batch/v2alpha1/defaults_test.go

```diff
@@ -20,179 +20,58 @@ import (
 	"reflect"
 	"testing"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/api"
 	_ "k8s.io/kubernetes/pkg/api/install"
-	"k8s.io/kubernetes/pkg/api/v1"
 	_ "k8s.io/kubernetes/pkg/apis/batch/install"
 	. "k8s.io/kubernetes/pkg/apis/batch/v2alpha1"
 )
 
-func TestSetDefaultJob(t *testing.T) {
-	defaultLabels := map[string]string{"default": "default"}
+func TestSetDefaultCronJob(t *testing.T) {
 	tests := map[string]struct {
-		original     *Job
-		expected     *Job
-		expectLabels bool
+		original *CronJob
+		expected *CronJob
 	}{
-		"both unspecified -> sets both to 1": {
-			original: &Job{
-				Spec: JobSpec{
-					Template: v1.PodTemplateSpec{
-						ObjectMeta: metav1.ObjectMeta{Labels: defaultLabels},
-					},
+		"empty CronJob should default ConcurrencyPolicy and Suspend": {
+			original: &CronJob{},
+			expected: &CronJob{
+				Spec: CronJobSpec{
+					ConcurrencyPolicy: AllowConcurrent,
```

Comment:
I just realized we didn't specify default `ConcurrencyPolicy` of this field (although it's mentioned in types.go, it's not discoverable for users, see https://kubernetes.io/docs/api-reference/v1.5/#cronjobspec-v2alpha1). We can fix this in a follow-up PR.